CONTRIBUTING.md

```diff
@@ -2,7 +2,15 @@
 
 ## Introduction
 
-This documentation is intended to assist contributors to pgBackRest by outlining some basic steps and guidelines for contributing to the project. Coding standards to follow are defined in [CODING.md](https://github.com/pgbackrest/pgbackrest/blob/master/CODING.md). At a minimum, unit tests must be written and run and the documentation generated before submitting a Pull Request; see the [Testing](#testing) section below for details.
+This documentation is intended to assist contributors to pgBackRest by outlining some basic steps and guidelines for contributing to the project.
+
+Code fixes or new features can be submitted via pull requests. Ideas for new features and improvements to existing functionality or documentation can be [submitted as issues](https://github.com/pgbackrest/pgbackrest/issues). You may want to check the [Project Boards](https://github.com/pgbackrest/pgbackrest/projects) to see if your suggestion has already been submitted.
+
+Bug reports should be [submitted as issues](https://github.com/pgbackrest/pgbackrest/issues). Please provide as much information as possible to aid in determining the cause of the problem.
+
+You will always receive credit in the [release notes](http://www.pgbackrest.org/release.html) for your contributions.
+
+Coding standards are defined in [CODING.md](https://github.com/pgbackrest/pgbackrest/blob/master/CODING.md) and some important coding details and an example are provided in the [Coding](#coding) section below. At a minimum, unit tests must be written and run and the documentation generated before submitting a Pull Request; see the [Testing](#testing) section below for details.
```

Comment:
Perhaps another idea here. Would it be interesting to outline how to work in PR's ? Like "the reviewer will solve the comment once fixed,..."
I know it since you guided me on that path but perhaps it could be nice to write it here too ?

Reply:
> Perhaps another idea here. Would it be interesting to outline how to work in PR's ? Like "the reviewer will solve the comment once fixed,..."
> I know it since you guided me on that path but perhaps it could be nice to write it here too ?

Agreed, and there is a card to address this in addition to a few other things https://github.com/pgbackrest/pgbackrest/projects/2#card-54174338 - but we are not sure if that will make it into this PR or be worked on in a separate PR.

Reply:
Agreed, let's keep that for a separate PR.